src/main/java/WayofTime/alchemicalWizardry/client/renderer/RenderHelper.java

```diff
@@ -114,18 +114,20 @@ private static void renderLPHUD(Minecraft mc, int lpAmount, int maxAmount) {
 
         ResourceLocation test2 = new ResourceLocation("alchemicalwizardry", "textures/gui/container1.png");
         GL11.glColor4f(1, 0, 0, 1.0F);
-        mc.getTextureManager().bindTexture(test2);
+		GL11.glEnable(3042);
```

Comment:
Please, use constants defined in `GL11` class for better readability:
```suggestion
		GL11.glEnable(GL11.GL_BLEND);
```
For blending you should also specify function via `GL11.glBleendFunc`.

Reply:
Also, run `./gradlew spotlessApply` when you're done to fix formatting.